cmd/createtree/main_test.go

```diff
@@ -105,27 +114,34 @@ func TestCreateTree(t *testing.T) {
 			wantErr:  true,
 		},
 		{
-			desc:      "createErr",
-			createErr: errors.New("create tree failed"),
-			wantErr:   true,
+			desc:         "createErr",
+			expectCreate: 4,
```

Comment:
This seems quite brittle - if the logic of the backoff or the context timeout changes this is likely to start failing in unexpected ways.
So, how about if an `err` is set for the corresponding function, we implicitly set up the expect as `MinTimes(1)` or similar?

Reply:
sounds good